vcluster/reference/migrations/loft-cli-vcluster-cli-migration.mdx

```diff
@@ -0,0 +1,110 @@
+| `loft use cluster`                | `vcluster platform connect cluster`             |                                                                                                                                                                                                                                                                                                               |
+| `loft list clusters`              | `vcluster platform list clusters`               |                                                                                                                                                                                                                                                                                                               |
+| `loft vars cluster`               | `vcluster platform get current-cluster -o name` |                                                                                                                                                                                                                                                                                                               |
+
+### vcluster commands
+
+| loft CLI                          | vcluster CLI                                    | Comments                                                                                                                                                                                                                                                                                                      |
+|-----------------------------------|-------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `loft connect vcluster`           | `vcluster connect --manager=platform`           | Following flags have been **deprecated**:<br/>- `--kube-config`<br/>- `--kube-config-context-name`<br/>- `--update-current`<br/><br/>To obtain the kube config for the given virtual cluster issue the following:<br/>`vcluster connect myvcluster --print &amp;gt; /tmp/myvcluser-kube-config.yaml`                |
+| `loft create vcluster`            | `vcluster create --manager=platform`            | Following flags have been **removed**:<br/>- `--delete-after`<br/>- `--disable-direct-cluster-endpoint`<br/>- `--sleep-after`<br/>- `--space` (vCluster platform handles this transparently in the background)<br/><br/>Following flags have been **changed**:<br/>- `--version` was renamed to `--template-version` |
```

Comment:
Can we remove the `-` in front of the flags to keep the look cleaner? 

It seems cluttered where the `-` is coming up as a `-` instead of bullet point


So either remove the dashes or try using `*` to see if it will make the bullet points:
* `--flag`
* `--flag2`

<img width="326" alt="Screenshot 2024-06-04 at 9 22 52 AM" src="https://github.com/loft-sh/vcluster-docs/assets/11410997/db3c4f48-ffb6-42f9-a697-b30ac2325cbb">